### ISSUE TITLE
Pyro5 could provide pointers regarding classes needing exposing

### ISSUE THREAD

**Issue by pevogam** [irmen/Pyro5#97]
I often get errors like
```
File "/usr/lib/python3.12/site-packages/Pyro5/client.py", line 510, in __call__  │
  return self.__send(self.__name, args, kwargs) │
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ │
File "/usr/lib/python3.12/site-packages/Pyro5/client.py", line 275, in _pyroInvoke  │
  raise data  # if you see this in your traceback, you should probably inspect the remote traceback as well │
  ^^^^^^^^^^│
 Pyro5.errors.PyroError: remote object doesn't expose any methods or attributes. Did you forget setting @expose on them?
```
It can be really helpful if this mentioned an actual name of the remote object in order to speed up any debugging process.

**Comment by irmen** [comment]
Makes sense, somehow this error message didn't contain the Pyro URI , whereas other similar error messages already included it.

**Comment by irmen** [comment]
fixed in 9c2e42d50976c9a67e9f843779f14d6c83964e31
